### PR TITLE
[REVIEW]gquant 1.0.1

### PR DESCRIPTION
**Pull request by yidong72**
This release added 'gquant-plugin' entry point support. And it is a placeholder for gquantlab plugin for Jupyterlab2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-## [1.0](https://github.com/rapidsai/gQuant/tree/1.0) (2020-12-17)
+## [1.0.1](https://github.com/rapidsai/gQuant/tree/1.0.1) (2021-01-20)
 
-[Full Changelog](https://github.com/rapidsai/gQuant/compare/0.5...1.0)
+[Full Changelog](https://github.com/rapidsai/gQuant/compare/v1.0.0...1.0.1)
```
Comment (avolkov1) [inline CHANGELOG.md:5]: Will the version be tagged with or without "v" as `1.0.1` or as `v1.0.1`. If with "v" update the changelog to use "v1.0.1".

**Review by avolkov1 — COMMENTED** [review]
Looks good. Just clarify the version tag with or without "v".

File: CHANGELOG.md (hunk 1 above)
Comment (yidong72) [reply CHANGELOG.md]: good catch. I added the 'v'

**Comment by yidong72** [comment]
Let's make it consistent to always prepend a 'v' to the version number.

**Review by avolkov1 — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**